test/casbin.test.js

```diff
@@ -8,6 +8,7 @@ const proxyquire = require('proxyquire')
 const sinon = require('sinon')
 
 const plugin = require('../')
+const { Model, FileAdapter } = require('casbin')
```

Comment:
please move this together with the external requires at the top

Reply:
done